### ISSUE TITLE
[Issue]:  Can't use multiple ControlNet models at once

### ISSUE THREAD

**Issue by Manchovies** [vladmandic/automatic#2087]
### Issue Description

In A1111 what I'm used to being able to do is to choose two or more different ControlNet models and each with their own input image. In this version, I have it set in the settings so that I can have 2 CN models at once under the CN settings page, and I've restarted the server, but I can still only choose one model and input image. How do I fix this?

### Version Platform Description

Windows, latest release

### Backend

Original

### Model

SD 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by bjornlarssen** [comment]
Same issue. Colab, latest release.

**Comment by vladmandic** [comment]
Change issue to extension template. Marking as invalid until it's corrected.

**Comment by brunogcar** [comment]
for reference https://github.com/Mikubill/sd-webui-controlnet/issues/1977 

that is why there are those requirements, cos it is not related do SD NEXT code,  but ControlNet Extension

and it should be included on OP as it is required in instructions, but when there isnt already an issue reported on extension github, create one

**Comment by fbellgr** [comment]
Yes. I was about to file an issue on the settings resetting each time we send a new image to img2img, assuming it was part of all the Gradio problems we had.

But to be certain, I checked out ControlNet from a few days ago and launch again, skipping updates, and the issue was gone. So definitely glitches were introduced in a recent revision of the extension itself.
